openhtf/io/frontend/__main__.py

```diff
@@ -77,7 +77,9 @@
 
 
 class Station(mutablerecords.Record(  # pylint: disable=too-few-public-methods,no-init
-    'Station', [], {'station_id': UNKNOWN_STATION_ID, 'state': None})):
+    'Station', [], {'hostport': (None, None),
```

Comment:
i think host and port, either as a tuple or as two values, should be not default-constructable; it doesn't make sense to have a Station without a host/port because you are indexing them by host/port.